apps/tlon-mobile/src/screens/ChannelScreen.tsx

```diff
@@ -6,7 +6,7 @@ import type * as db from '@tloncorp/shared/dist/db';
 import * as store from '@tloncorp/shared/dist/store';
 import { handleImagePicked, useUploader } from '@tloncorp/shared/dist/store';
 import type { Story } from '@tloncorp/shared/dist/urbit';
-import { Channel, ChannelSwitcherSheet, View } from '@tloncorp/ui';
+import { Channel, ChannelSwitcherSheet, View } from '@tloncorp/ui/src';
```

Comment:
src

Reply:
Oops didn't mean to press add single comment lol.

Reply:
OUT:  _oops i forgot to undo replacing showDevTools with false_
IN: _oops i left in /src_